admin/src/api/post/content-types/post/lifecycles.js

```diff
@@ -162,7 +167,7 @@ function validateFields(result) {
     });
     throw error;
   }
-  if (result.author.connect.length == 0) {
+  if (!is_from_update && result.author.connect.length == 0) {
```

Comment:
Why do we need to check `is_from_update`, Why we are not getting author data on edit? Can you double check events.params.data?

Reply:
- no, as have told before, it's not having current author value in request.
- also, it's not fetching author value when we edit the post.
that's why it's no giving maybe

Reply:
Can you check, what is value of author if we don't set then on create ? Also check it on edit. Match them and take decision base on that instead of passing editing flag.

If above cases are not suitable then its fine to take this flag

Reply:
Create without author:
`author: { disconnect: [], connect: [] },`

Create with author:
`author: { disconnect: [], connect: [ [Object] ] },`

Update without selected new:
`author: { disconnect: [], connect: [] },`

Update without changing author(older selected author):
`author: { disconnect: [], connect: [] },`